core/src/components/range/range.tsx

```diff
@@ -502,7 +501,7 @@ export class Range implements ComponentInterface {
             part="bar-active"
           />
 
-          { renderKnob(isRTL, {
+          {renderKnob(isRTL(this.el), {
```

Comment:
Same thing here. The calls in this function can cached within the function context